tests/test_parse.py

```diff
@@ -352,13 +352,6 @@ def test_skip_line(self):
     ###########################################################################
     # Status code issues
     ###########################################################################
-    def test_status_disallowed(self):
```

Comment:
If we were to accept this change, we'd want an explicit test showing that 403 is allowed instead of just removing the test.